samples/azure-quantum/chemistry/Hamiltonian/Approximate Energy using Jordan-Wigner transformation.ipynb

```diff
@@ -0,0 +1,813 @@
+      "   Job ID: 668a44cd-b79d-4685-a376-befdfe924cf7\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.simulator...\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.simulator...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 8\n",
+      "   Job ID: 82b24b66-6a80-46ab-9a57-f1c03ea09877\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.simulator...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 9\n",
+      "   Job ID: e8d9496f-cc03-4136-8eb8-eecbb210489c\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.simulator...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 12\n",
+      "   Job ID: 34631dd5-e6e2-4d68-adb1-dd933ce3d864\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.simulator...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 13\n",
+      "   Job ID: 75448e3d-aadc-4970-8470-2b19586ada1e\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.simulator...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 16\n",
+      "   Job ID: 16156050-e4ac-4c11-8ae5-ec2dc0086d0b\n",
+      "Submitting Microsoft.Quantum.Chemistry.Hamiltonian.GetHamiltonianTermH2 to target ionq.simulator...\n",
+      "Job successfully submitted for 1000 shots.\n",
+      "   Job name: Hamiltonian term 17\n",
+      "   Job ID: fff66844-b07b-46ec-b221-0442d19d51c6\n"
+     ]
+    }
+   ],
+   "source": [
+    "jobs = [\n",
+    "    qsharp.azure.submit(GetHamiltonianTermH2, nOp=n_op, shots=1000, jobName=f\"Hamiltonian term {n_op}\") \n",
+    "    for n_op, coeff in enumerate(coeffs) if coeff != 0.0\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "9e6f51d5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = [qsharp.azure.output(j.id) for j in jobs]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "002adcf4",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{'1': 1.0},\n",
+       " {'1': 1.0},\n",
+       " {'0': 1.0},\n",
+       " {'0': 1.0},\n",
+       " {'0': 1.0},\n",
+       " {'1': 1.0},\n",
+       " {'1': 1.0},\n",
+       " {'1': 1.0},\n",
+       " {'1': 1.0},\n",
+       " {'0': 1.0},\n",
+       " {'0': 0.5, '1': 0.5},\n",
+       " {'0': 0.5, '1': 0.5},\n",
+       " {'0': 0.5, '1': 0.5},\n",
+       " {'0': 0.5, '1': 0.5}]"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results"
```

Comment:
Can combine this cell with the previous one, I generally try to avoid output-less cells